docs/plugins/minify-simplify.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: docs
+title: Minify simplify
+description: This plugin will transform code in mainly two ways - 1) Reduce as much statements as possible into expressions, and 2) Make expressions as uniform as possible for better compressibility
+permalink: /docs/plugins/minify-simplify/
+package: babel-plugin-minify-simplify
+---
+
+{% include package_readme.html from="babili" %}
```

Comment:
Need to add an extra line between the text and the code block, since markdownify behaves differently than GH flavored markdown.